Shared/BetterColorPicker.cs

```diff
@@ -71,6 +72,14 @@ private void Awake()
                 "When using default saturation filter the game colors are different than actual colors. " +
                 "Use this setting to adjust the color you capture to make it look correct under the saturation filter. " +
                 "If you do not use the saturation filter, disable this option to get the true color.");
+
+            KKAPI.Maker.MakerAPI.MakerFinishedLoading += (sender, e) => { MakePickerDraggable(); };
+        }
+
+        private void MakePickerDraggable()
+        {
+            var go = GameObject.Find("CustomScene/CustomRoot/FrontUIGroup/CustomUIGroup/CvsColor/Top");
+            go?.AddComponent<DragObject>();
```

Comment:
```suggestion
            if (go) go.GetOrAddComponent<DragObject>();
```
Prevent potentially adding duplicate copies in KKS.